### PR TITLE
Normalize credentials for all providers

### PR DESCRIPTION
**Pull request by tjanczuk**

### CODE REVIEW

File: src/provider/index.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import * as slack from './slack';
+import * as oauth from './oauth';
 
 export * as slack from './slack';
+export * as oauth from './oauth';
 
 export type INative = slack.INative;
```
Comment (bennbollay) [inline src/provider/index.ts:7]: Include an oauth.INative here that publishes as any if necessary.

Comment (tjanczuk) [reply src/provider/index.ts:7]: I am not sure what you mean by "include". This module exports a single INative type that is hardwired to that of Slack.

Comment (bennbollay) [reply src/provider/index.ts:7]: export type INative = slack.INative | oauth.INative;

Comment (tjanczuk) [reply src/provider/index.ts]: ok added `| any`

File: src/provider/index.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import * as slack from './slack';
+import * as oauth from './oauth';
 
 export * as slack from './slack';
+export * as oauth from './oauth';
 
-export type INative = slack.INative;
+export type INative = slack.INative | any;
```
Comment (bennbollay) [inline src/provider/index.ts:7]: Please stay consistent with the other code and export an `oauth.INative` type.

Comment (tjanczuk) [reply src/provider/index.ts]: done

**Review by bennbollay — APPROVED** [review]

**Pull request merged**